cmd/node-observability-agent/main.go

```diff
@@ -46,7 +48,7 @@ func main() {
 	log.SetLevel(lvl)
 	log.Infof("Starting %s at log level %s", ver.MakeVersionString(), *logLevel)
 
-	checkParameters(*tokenFile, node, *storageFolder, *crioSocket, *caCertFile)
```

Comment:
Any reason we are ditching this check? (AFAIK seems a little extra defensive that is it)

Reply:
The token was read twice: in `checkParameters` and right after. Now it gets read only once and it something goes wrong we'll see it anyway.

Reply:
Ahh got it, it was outside in the `main` func.